cardano-api/src/Cardano/Api/InMode.hs

```diff
@@ -1,7 +1,21 @@
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE GADTs #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE DataKinds                  #-}
```

Comment:
Why is your formatter aligning the pragmas?

Reply:
Fixed!